src/natcap/invest/utils.py

```diff
@@ -538,6 +541,30 @@ def build_lookup_from_csv(
     return lookup_dict 
 
 
+def read_csv_to_dataframe(path, to_lower=True, strip=True, **kwargs):
+    """
+    Wrapper around ``pandas.read_csv`` that standardizes the column names by
+    stripping leading/trailing whitespace and making all lowercase.
+    This helps avoid common errors caused by user-supplied CSV files with
+    column names that don't exactly match the specification.
+
+    Args:
+        path (string): path to a CSV file
+        to_lower (bool): if True, convert all column names to lowercase
+        strip (bool): if True, strip leading/trailing whitespace from all
+            column names
+        **kwargs: any kwargs that are valid for ``pandas.read_csv`` or
+            ``pandas.read_excel``
```

Comment:
This mentions `read_excel` as well ... was the intent here to also add excel support?

Reply:
Oh yeah... after seeing that excel was an option for the HRA model, I tried adding it, but I think it's better to handle the HRA model inputs separately unless we decide that all models should allow excel files

Reply:
👍